cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2661,6 +2661,31 @@
     </description>
   </property>
 
+  <property>
+    <name>messaging.spanner.properties.publish.batch.size</name>
+    <value>50</value>
+    <description>
+      The default max number of messages in each publish batch for spanner messaging service.
+    </description>
+  </property>
+
+  <property>
+    <name>messaging.spanner.properties.publish.batch.timeout.millis</name>
+    <value>50</value>
+    <description>
+      The default maximum waiting time before a batch is published by spanner messaging service.
+    </description>
+  </property>
+
+  <property>
+    <name>messaging.spanner.properties.publish.delay.millis</name>
```

Comment:
I would not call this delay. AFAICT, this is polling frequency while either batch.size is met or batch.timeout is met. How about `messaging.spanner.properties.publish.batch.poll.millis` ?